py3status/modules/fedora_updates.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-The number of package updates pending for a Fedora Linux installation.
+Display number of pending updates for Fedora Linux.
```

Comment:
I like to keep `number of pending updates`. I already changed many other things. I think we did great! Time for break? Bbl. 👍